pmm-app/src/pmm-dbaas/DBaaS.messages.ts

```diff
@@ -0,0 +1,35 @@
+export const Messages = {
+  tabs: {
+    manageDB: 'Manage DB Cluster',
+    kubernetes: 'Kubernetes Cluster Inventory',
+  },
+  kubernetes: {
+    deleteAction: 'Delete',
+    addAction: 'Add new K8s Cluster',
```

Comment:
Change to "Add new Kubernetes Cluster" (I know it's not what is on the design but makes it more consistent)